web/alt_text_manager.js

```diff
@@ -66,11 +75,26 @@ class AltTextManager {
     this.#overlayManager.register(dialog);
   }
 
+  get #elements() {
```

Comment:
Given that this is repeatedly accessed, maybe make this a shadowed getter?

Reply:
One time when displaying the dialog and an other one when removing the pointerDown listeners.
Do you think we should consume some memory for this thing ?